src/nupic/os/Directory.cpp

```diff
@@ -23,7 +23,9 @@
 /** @file 
 */
 
-#include <unistd.h>
+#if !defined(NTA_OS_WINDOWS)
+  #include <unistd.h>
```

Comment:
> Won't you get compiler errors on Windows since those functions are undefined now?

Not because `windows.h` is included here: https://github.com/david-ragazzi/nupic.core/blob/update_source_to_windows/src/nupic/os/Directory.cpp#L38

By the way we could move `#include <unistd.h>` to `#else` block of `#if defined(NTA_OS_WINDOWS)`.